### DISCUSSION TITLE
[Planned upcoming .NET 6/Azure Function work] - AWS Lambda support?

### DISCUSSION THREAD

**Discussion by kieran-osgood** [ChilliCream/hotchocolate#4234] [Ideas]
Morning!

In the [HotChocolate Project Card - .NET 6 #55723117](https://github.com/ChilliCream/hotchocolate/projects/28#card-55723117) and also [HotChocolate Project Card - azure function integrations #68302440](https://github.com/ChilliCream/hotchocolate/projects/28#card-68302440) I see there is planned work for supporting .Net 6 and the azure functions environment.

AWS previously got support for .NET 5 after about a month, so hopefully by EOY we could be looking at native support for .NET 6 on AWS Lambda.

I wanted to enquire if there is any interest/thoughts from the HotChocolate team (or the HC community, if the latter maybe 👍 this if this is of interest to you) towards supporting AWS officially (not entirely sure what this would entail if I'm honest - just wanted to open discussion and find out thoughts/if theres potential in it).

**Comment by michaelstaib** [comment]
With V13 we will add official support for AzF. AWS Lambda had support of running a proper ASP.NET core host in them which enables full support of everything just out of the box.
